src/App.js

```diff
@@ -1,7 +1,54 @@
 import "./App.css";
+import {
+  BrowserRouter as Router,
+  NavLink,
+  Switch,
+  Route,
+} from "react-router-dom";
+
+import Home from "./components/Home";
+import CharactersList from "./components/CharactersList";
+import CharacterDetails from "./components/CharacterDetails";
 
 function App() {
-  return <div className="App">Rick & Morty</div>;
+  return (
+    <Router>
+      <div className="App">
+        <header className="header">
+          <nav className="nav">
+            <img
+              className="navImg"
+              alt="navImg"
+              src="https://i.kym-cdn.com/photos/images/original/001/277/665/eaa.jpg"
+            ></img>
+            <div className="navLinks">
+              <NavLink exact to="/">
+                Home
+              </NavLink>
+              <NavLink to="/charactersList">Characters List</NavLink>
+            </div>
+          </nav>
+        </header>
+        <main className="main">
```

Comment:
This would probably better placed in the different components, which get rendered depending on the route